src/main/java/org/folio/inventory/dataimport/handlers/matching/loaders/AuthorityLoader.java

```diff
@@ -52,4 +59,13 @@ protected String addCqlSubMatchCondition(DataImportEventPayload eventPayload) {
   protected String mapEntityToJsonString(Authority authority) {
     return Json.encode(authority);
   }
+
+  @Override
+  protected String mapEntityListToIdsJson(List<Authority> authorityList) {
```

Comment:
General information.

It would be clearer if the method returned something like a `JsonObject` or similar so that it is enforced that a correct JSON object is returned. Since it is just a string, it could be anything! You are overriding a method so i dont expect any change to the API!

Reply:
agree, definitely method declaration with JsonObject would enforce to return correct json string representation. Just, in fact, the method is used only to get json string representation, so to avoid unused converting to JsonObject and then deserialization to string a less strict return type signature is used as a compromise.